mifosng-android/src/main/java/com/mifos/mifosxdroid/login/LoginActivity.java

```diff
@@ -240,16 +264,7 @@ public void onLoginClick(Button button) {
         // Saving port
         PrefManager.setPort(et_port.getEditableText().toString());
         // Updating Services
-        if (!PrefManager.getInstanceDomain().equals(BaseUrl.API_ENDPOINT)) {
-            BaseApiManager.createService();
-        }
-
-        login(false);
-    }
-
-    private void login(boolean shouldByPassSSLSecurity) {
-        if (!validateUserInputs())
-            return;
+        BaseApiManager.createService();
```

Comment:
Yeah, I got it, Sorry I have missed one case, Thanks go ahead for the rest of things.